src/main/java/com/github/javafaker/BrazilianDocuments.java

```diff
@@ -0,0 +1,216 @@
+package com.github.javafaker;
+
+import java.util.InputMismatchException;
+
+public class BrazilianDocuments {
+
+    private int randomiza(int n) {
+      return (int) (Math.random() * n);
+    }
+
+    private int mod(int dividendo, int divisor) {
+      return (int) Math.round(dividendo - (Math.floor(dividendo / divisor) * divisor));
+    }
+
+    public String cpf(boolean withPunctuation) {
```

Comment:
I don't understand the requirements for the number being generated. Could you add some documentation please? 

Also if it's not too much to ask, could you translate it to english? It would make it easier to maintain